File: cpp/bc1020.cpp
#include <iostream>

using namespace std;

int main()
{
	int valIn, day, mounth, year, i;
	
	day = mounth = year = 0;
	
	cin >> valIn;
	
	for(int i = 1; i <= valIn; i++)
	{
		day++;
		
		if(day == 30)
		{
			mounth++;
			day = 0;
		}
		if(mounth == 12)
		{
			year++;
			mounth = 0;
			day = -5;
		}	
	}
	
	cout << year << " anos(s)\n" << mounth << " mes(es)\n" << day << " dia(s)" << endl;
}

File: python/bc1020.py
valIn = int(input())

year = int(valIn / 360)
mounth = int((valIn % 365) / 30)

if(((valIn%360)%30)%5 == 0): day = (valIn%365) % 30
else : day = ((valIn%360)%30) - 5 * year

print(f"{year} ano(s)\n{mounth} mes(es)\n{day} dia(s)")